flink-runtime/src/test/java/org/apache/flink/runtime/rpc/RpcEndpointTest.java

```diff
@@ -20,7 +20,7 @@
 
 import org.apache.flink.api.common.time.Time;
 import org.apache.flink.runtime.akka.AkkaUtils;
-import org.apache.flink.runtime.concurrent.FutureUtils;
+import org.apache.flink.runtime.concurrent.akka.AkkaFutureUtils;
```

Comment:
Replacing `FutureUtils` leads to a compilation error: The test uses `FutureUtils.waitForAll`.